net-misc/frr/files/frr-9.0-bug-913242.patch

```diff
@@ -0,0 +1,173 @@
+diff --git a/bgpd/bgp_attr.c b/bgpd/bgp_attr.c
+index 60e41ac1cc83..7badc6a17cbf 100644
```

Comment:
Could you please add a link to upstream PR/commit where these are taken from?

Reply:
I removed 9.0 as 9.0.2 is needed for new CVEs